src/seedu/addressbook/AddressBook.java

```diff
@@ -31,1137 +32,1334 @@
  */
 
 /**
- * This class is used to maintain a list of person data which are saved
- * in a text file.
+ * This class is used to maintain a list of person data which are saved in a
+ * text file.
  **/
 public class AddressBook {
 
-    /**
```

Comment:
Don't indent the whole file next time. Or not it would be hard to spot the difference. :)